app/src/main/java/com/vitorpamplona/amethyst/service/model/zaps/ZapAmount.kt

```diff
@@ -0,0 +1,14 @@
+package com.vitorpamplona.amethyst.service.model.zaps
+
+import java.math.BigDecimal
+
+interface ZapAmountInterface {
+    fun total(): BigDecimal?
+}
+
+class ZapAmount(var amount: BigDecimal?) : ZapAmountInterface {
```

Comment:
I am not sure if this is needed anymore. It was an idea before, but I think I can safely remove it. I will try removing it to keep the code simpler.